legal-api/tests/unit/models/test_corp_type.py

```diff
@@ -0,0 +1,58 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests to assure the CorpType Model.
+
+Test-Suite to ensure that the CorpType Model is working as expected.
+"""
+
+from legal_api.models import Business, CorpType
+
+
+def test_corp_type_json(session):
+    """Assert the json format of corp type."""
+    corp_type = CorpType.find_by_id(Business.LegalTypes.BCOMP.value)
+    corp_type_json = {
+        'corp_type_cd': 'BEN',
+        'colin_ind': 'Y',
+        'corp_class': 'BC',
+        'short_desc': 'BENEFIT COMPANY',
+        'full_desc': 'BC Benefit Company',
+        'legislation': 'BC Business Corporations Act'
+    }
+    assert corp_type_json == corp_type.json
+
+
+def test_find_corp_type_by_id(session):
+    """Assert that the method returns the corp type matching the id."""
+    corp_type = CorpType.find_by_id(Business.LegalTypes.BCOMP.value)
+    corp_type_json = {
```

Comment:
I think hardcoding the values in `corp_type_json` would make more sense for your testing assertions.